### PR TITLE
Feature/tensorflow

### PR DESCRIPTION
**Pull request by datelier**
Tensorflow for Go

### Description:
This PR adds a feature for Tensorflow.

### Environment:

- Golang Version: 1.13.5
- Docker Version: 19.03.5
- Kubernetes Version: 1.16.3
- NGT Version: 1.8.4

### Types of changes:

- [ ] Bug fix [type/bug]
- [x] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensure all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: internal/core/converter/tensorflow/option.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package tensorflow provides implementation of Go API for extract data to vector
+package tensorflow
+
+import (
+	tf "github.com/tensorflow/tensorflow/tensorflow/go"
```
Comment (kmrmt) [inline internal/core/converter/tensorflow/option.go]: unused import

Comment (kpango) [reply internal/core/converter/tensorflow/option.go]: you must be next generation go compiler right?

Comment (kmrmt) [reply internal/core/converter/tensorflow/option.go]: nooooooooo! i'm not sure about go...

Comment (datelier) [reply internal/core/converter/tensorflow/option.go]: fixed

**Pull request merged**